### PR TITLE
Update Readme with resultID

### PR DESCRIPTION
**Pull request by adetch**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -144,5 +144,5 @@ constructorIO.trackPurchase(customerIDs: ["123-AB", "456-CD"])
 constructorIO.trackBrowseResultsLoaded(filterName: "Category", filterValue: "Snacks", resultCount: 674)
 
 // Track when a browse result is clicked
-constructorIO.trackBrowseResultClick(filterName: "Category", filterValue: "Snacks", customerID: "7654321-BA", resultPositionOnPage: 4)
+constructorIO.trackBrowseResultClick(filterName: "Category", filterValue: "Snacks", customerID: "7654321-BA", resultPositionOnPage: 4, resultID: "179b8a0e-3799-4a31-be87-127b06871de2")
```
Comment (crgee1) [inline README.md:147]: there is a sectionName argument before resultID, so we should add the sectionName as well so something like `sectionName: "Products", resultID: "179b8a0e-3799-4a31-be87-127b06871de2"`

File: README.md [hunk 2]
```diff
@@ -128,7 +128,7 @@ constructorIO.trackSearchSubmit(searchTerm: "toothpicks", originalQuery: "tooth"
 constructorIO.trackSearchResultsLoaded(searchTerm: "tooth", resultCount: 789)
 
 // Track when a search result is clicked
-constructorIO.trackSearchResultClick(itemName: "Fashionable Toothpicks", customerID: "1234567-AB", searchTerm: "tooth")
+constructorIO.trackSearchResultClick(itemName: "Fashionable Toothpicks", customerID: "1234567-AB", searchTerm: "tooth", resultID: "179b8a0e-3799-4a31-be87-127b06871de2")
```
Comment (crgee1) [inline README.md:131]: there is a sectionName argument before resultID, so we should add the sectionName as well so something like `sectionName: "Products", resultID: "179b8a0e-3799-4a31-be87-127b06871de2"`

Comment (adetch) [reply README.md:131]: Ah, I guess we missed the section name for the original docs?

File: README.md [hunk 3]
```diff
@@ -115,7 +115,7 @@ Three types of these events exist:
 constructorIO.trackInputFocus(searchTerm: "")
 
 // Track when the user selects an autocomplete suggestion
-constructorIO.trackAutocompleteSelect(searchTerm: "toothpicks", originalQuery: "tooth", sectionName: "Search Suggestions")
+constructorIO.trackAutocompleteSelect(searchTerm: "toothpicks", originalQuery: "tooth", sectionName: "Search Suggestions", resultID: "179b8a0e-3799-4a31-be87-127b06871de2")
```
Comment (crgee1) [inline README.md:118]: Theres a `group` argument before resultID of type `CIOGroup`, so a full example would like something like 

`constructorIO.trackAutocompleteSelect(searchTerm: "toothpicks", originalQuery: "tooth", sectionName: "Search Suggestions", group: CIOGroup(displayName: "groupName1", groupID: "groupID2", path: "path/to/group"), resultID: "179b8a0e-3799-4a31-be87-127b06871de2")`

File: README.md (hunk 2 above)
Comment (adetch) [reply README.md]: e8ab7e4

File: README.md (hunk 1 above)
Comment (adetch) [reply README.md]: e8ab7e4

File: README.md (hunk 3 above)
Comment (adetch) [reply README.md]: 1b84b16

**Review by crgee1 — APPROVED** [review]
looks good

**Pull request merged**